frontend/src/core/hooks/usePresignedURL.js

```diff
@@ -15,33 +16,33 @@ const usePresignedURL = ({
 
   const getFromPresignedURL = async () => {
     let request_parameters = {
-      url: presignedRequest.url,
+      url: url,
       // You can uncomment this to use mockupsLibrary in development
       // url: `https://example.com/s3`,
       headers: {},
       method: "GET",
     };
 
-    if ("headers" in presignedRequest) {
-      Object.keys(presignedRequest.headers).map((key) => {
-        request_parameters["headers"][key] = presignedRequest.headers[key];
+    if (headers != undefined) {
```

Comment:
why not `if (headers)`? 
or more explicit `if(Object.keys(headers))`

Reply:
Explicit way for me not obvious how it's working with undefined.
But if headers looks correct.

Reply:
Done as `if (headers)`